tuxemon/core/states/combat/combat_menus.py

```diff
@@ -67,6 +67,11 @@ def on_menu_selection(self, menuitem):
     def open_swap_menu(self):
         def swap_it(menuitem):
             monster = menuitem.game_object
+            if monster in self.game.get_state_name('CombatState').active_monsters:
+                tools.open_dialog(self.game, ["Monster is already in play"])
```

Comment:
Instead of hardcoding English dialogs, we should ideally create an entry in [resources/db/locale/en_US.json](https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/resources/db/locale/en_US.json) and use the `translate` function like used here:

https://github.com/Tuxemon/Tuxemon/blob/development/tuxemon/core/states/persistance/save_menu.py#L90